lib/l10n/l10n.dart

```diff
@@ -127,6 +128,35 @@ class Language {
   String toString() => locale.toLanguageTag();
 }
 
+String formattedFileSize(int? bytes) {
```

Comment:
1. Documentation should be added:

> - the code needs to be covered with documentation following the rules of Effective Dart: Documentation;

2. Perhaps writing an extension over `int`egers would be a cleaner approach?

Reply:
Of course, I can use extension if you'd like to.

Typically, I don't use extension functions whenever I don't see an explicit reason to do so (they can get out of control quite easily).

Reply:
and I've added the doc string

Reply:
@Gieted,

> they can get out of control quite easily

Do you mean the possible [extension conflicts](https://dart.dev/language/extension-methods#api-conflicts)? This can be solved with more strict and granular extensions - for example, you may define one over `StorageFile` instead of `int` - this will narrow down the possible overlaps. Or is it something else?

Global functions and global variables have a set of downsides we may discuss. For example, extensions encapsulate certain methods over concrete types, whitest global methods are just there in the scope. Also this file, `l10n.dart`, has a series of extensions in it. It may raise a certain level of chaos by putting different approaches in the same file.

Reply:
Ok, I'll use the extension method